docs/installation.rst

```diff
@@ -5,14 +5,16 @@
 Installation
 ============
 
-To install |project_name|, you need `Python 3`_ and `pip`_ installed. You
-can then either
-`download it directly
+To install Borg-Import, you need `Python 3` and `pip` installed. You
+can then either `download borg-import directly
 <https://github.com/borgbackup/borg-import/archive/master.zip>`_
-or clone it from github_.
-Extract the content, open a terminal in its root folder and
-execute the following to install it via pip:
-:code:`pip install --user -e .`
+or clone it from `github<https://github.com/borgbackup/borg-import>`_.
+Unzip the downloaded .zip if necessary.
+Open a terminal in the borg-import directory and execute the following to 
+install the program via pip: :code:`pip install --user -e .`
 
 If you have */home/user/.local/bin/* in your ``PATH`` variable,
-you can then start using |project_name|.
+you can then start using Borg-Import.
+Otherwise, you will need to add *.local/bin/* to your ``PATH`` via adding
+``export PATH=$PATH:/home/user/.local/bin/`` to your .bash\_profile file.
```

Comment:
that is bash specific, not everybody uses bash, especially not on the non-linux platforms borg and python also support.